types/.gitignore

```diff
@@ -1 +1,10 @@
-/lib/
+# Don't track build artifacts
+**/*.js
+**/*.d.ts
+hiring/schemas
```

Comment:
Added it to ignore the `role.schema.json` which is moved into there because its imported by `src/hiring/index.ts`

```
import * as role_schema_json from './schemas/role.schema.json'
```

I will be more explicit and make it ignore that exact file.